scene/3d/sprite_3d.cpp

```diff
@@ -840,7 +840,7 @@ void AnimatedSprite3D::_draw() {
 	if (is_centered())
 		ofs -= s / 2;
 
-	Rect2i dst_rect(ofs, s);
+	Rect2i dst_rect(0, 0, s.width, s.height);
```

Comment:
Since `ofs` is no longer used here, the code would be easier to read if you moved the definition of `ofs` right before `final_rect.pos += ofs;`.